src/titiler/core/titiler/core/factory.py

```diff
@@ -885,6 +883,16 @@ def wmts(
             else:
                 supported_crs = tms.crs.srs
 
+            layers = [
+                {
+                    "title": src_path if isinstance(src_path, str) else "TiTiler",
+                    "name": "default",
+                    "tiles_url": tiles_url,
+                    "query_string": urlencode(qs, doseq=True) if qs else None,
+                    "bounds": bounds,
+                },
+            ]
```

Comment:
add layers to WMTS response 

this will be use for STAC renders but maybe also for multiple variable support in XARRAY